### PR TITLE
CSM Docs is having dead links

### PR DESCRIPTION
**Pull request by WilsonRadadia20**
# Description
Some of the links in CSM Docs are throwing 404 error.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| dell/csm#1289 |

# Checklist:

- [ ] Have you run a grammar and spell checks against your submission?
- [x] Have you tested the changes locally?
- [x] Have you tested whether the hyperlinks are working properly?
- [ ] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

File: content/docs/authorization/configuration/powerflex/_index.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ Given a setup where Kubernetes, a storage system, and the CSM for Authorization
 
     **Helm**
 
-    Refer to the [Install the Driver](../../../deployment/helm/drivers/installation/powerflex/#install-the-driver) section to edit the parameters in `samples/config.yaml` to configure the driver to communicate with the CSM Authorization sidecar.
+    Refer to the [Install the Driver](../../../deployment/helm/drivers/installation/powerflex.md/#install-the-driver) section to edit the parameters in `samples/config.yaml` to configure the driver to communicate with the CSM Authorization sidecar.
```
Comment (shanmydell) [inline content/docs/authorization/configuration/powerflex/_index.md:58]: @WilsonRadadia20 : why this change is required, we should not add extensions in hugo link

**Review by shanmydell — CHANGES_REQUESTED** [review]

File: content/docs/authorization/configuration/powerflex/_index.md (hunk 1 above)
Comment (WilsonRadadia20) [reply content/docs/authorization/configuration/powerflex/_index.md]: I have removed the extensions.

**Review by shanmydell — APPROVED** [review]

**Comment by shanmydell** [comment]
looks good to me

**Review by gallacher — APPROVED** [review]
